script/create_gibs_users.rb

```diff
@@ -14,12 +14,10 @@
       }
     }
     puts "account to be created: #{args}"
-    user = Card.create! args
+    Card.create! args
     puts "activating #{row[:name]}"
     status_card = Card["#{row[:name]}+*account+*status"]
     status_card.update_attributes! content: 'active', silent_change: true
-
-    Card::Mailer.perform_deliveries = true
-    user.account.send_welcome_email
   end
 end
+Card::Mailer.perform_deliveries = true
```

Comment:
fwiw, this is probably unnecessary, because the setting is specific to the process running this script.